core/trino-main/src/main/java/io/trino/server/PluginManager.java

```diff
@@ -74,6 +62,7 @@
 
     private static final Logger log = Logger.get(PluginManager.class);
 
+    private final PluginsProvider pluginLoader;
```

Comment:
`pluginsProvider`? or rename the ctor arg

did you mean to name the class differently?

Reply:
Thanks, I missed updating this one